lib/UsersStack.js

```diff
@@ -0,0 +1,154 @@
+const cdk = require('@aws-cdk/core');
+const dynamodb = require('@aws-cdk/aws-dynamodb');
+const apigateway = require('@aws-cdk/aws-apigateway');
+const lambda = require('@aws-cdk/aws-lambda');
+const UserModel = require('../src/schema/UserModel');
+
+class UsersStack extends cdk.Stack {
+    constructor(scope, id, props) {
+        super(scope, id, props);
+
+        const usersStack = this;
+        const { stackName } = props;
+
+        // create users table
+        const usersTable = new dynamodb.Table(usersStack, 'Users', {
+            partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+            sortKey: { name: 'sk', type: dynamodb.AttributeType.STRING },
+            billingMode: dynamodb.BillingMode.PAY_PER_REQUEST,
+            removalPolicy: cdk.RemovalPolicy.DESTROY, // allow table delete
```

Comment:
For sure.  This line is **not** be desired in a production setting as it would allow a delete of the table.  I left it there only so you could fully remove the stack upon a 'cdk destroy' during this demo.